kola/harness.go

```diff
@@ -358,8 +357,21 @@ func getClusterSemver(pltfrm, outputDir string) (*semver.Version, error) {
 	if err != nil {
 		return nil, fmt.Errorf("parsing /etc/os-release: %v: %s", err, stderr)
 	}
+	ver := strings.Split(string(out), "+")[1]
```

Comment:
This changed from `=` to `+`.  Was that intentional?

Reply:
It was not, typo'd.